support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/Pac4jWebflowConfigurer.java

```diff
@@ -30,32 +34,48 @@
  */
 public class Pac4jWebflowConfigurer extends AbstractCasWebflowConfigurer {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(Pac4jWebflowConfigurer.class);
+    
     private final Action saml2ClientLogoutAction;
+    private final Action ignoreServiceRedirectForSamlSloAction;
+    private final TerminateSessionAction terminateSessionAction;
+    private final Action limitedTerminateSessionAction;
     
     public Pac4jWebflowConfigurer(final FlowBuilderServices flowBuilderServices, 
                                   final FlowDefinitionRegistry loginFlowDefinitionRegistry,
                                   final FlowDefinitionRegistry logoutFlowDefinitionRegistry,
-                                  final Action saml2ClientLogoutAction, final ApplicationContext applicationContext,
+                                  final Action saml2ClientLogoutAction,
+                                  final Action ignoreServiceRedirectForSamlSloAction,
+                                  final TerminateSessionAction terminateSessionAction,
```

Comment:
Change type to action.

Reply:
Sorry, is this needed? I call a method on the action, if I change it into Action, I will have to typecast back to to the desired type. Isn't it better to have the proper type?